### PR TITLE
Add json support for SPV_EXT_fragment_invocation_density

### PR DESCRIPTION
**Pull request by dgkoch**
- which aliases the SPV_NV_shading_rate functionality

### CODE REVIEW

File: include/spirv/unified1/spirv.core.grammar.json [hunk 1]
```diff
@@ -6369,18 +6369,32 @@
           "extensions" : [ "SPV_NV_fragment_shader_barycentric" ],
           "version" : "None"
         },
+        {
+          "enumerant" : "FragSizeEXT",
```
Comment (dneto0) [inline include/spirv/unified1/spirv.core.grammar.json]: Putting the EXT names first will cause the disassembler to prefer the EXT names when outputting.  That's a good choice.  The only impact would be needing an update to SPIRV-Tools tests (if any have been made for the corresponding NV extension.)

File: include/spirv/unified1/spirv.core.grammar.json [hunk 2]
```diff
@@ -7280,7 +7294,14 @@
           "enumerant" : "ShadingRateNV",
           "value" : 5291,
           "capabilities" : [ "Shader" ],
-          "extensions" : [ "SPV_NV_shading_rate" ],
+          "extensions" : [ "SPV_NV_shading_rate", "SPV_EXT_fragment_invocation_density" ],
+          "version" : "None"
+        },
+        {
```
Comment (dneto0) [inline include/spirv/unified1/spirv.core.grammar.json]: I'd put the FragmentDensityEXT spelling first, so swap this element with the previous.  That would match the other enums.

**Comment by dgkoch** [comment]
Updated MR with requested changes.

**Comment by johnkslang** [comment]
Thank you @dneto0. I'll pull this in here and through to the one that generates the spec.

**Pull request merged**